### PR TITLE
Populate metadata from _cal to _i2d in image2 pipeline

### LINKED ISSUE #1449: Resampled (i2d) product from calwebb_image2 has nearly empty header
The resampled products for individual exposures coming out of the calwebb_image2 pipeline have nearly empty primary headers. Only the dozen or so keywords containing a record of the drizzle params, and a few others calculated by the resample step (e.g. EXPSTART, EXPEND) are present. I imagine this is due to the fact that the output DrizProductModel is created from scratch and a .update is never done on it to copy the meta data from the input file.

The resample code appears to have the param `blendheaders` hardwired to True, so I guess that means it's being called on the output DrizProductModel, but what does it do when there's only 1 input image? In the case of a single input, either `blendheaders` should be turned off and you just call output.update(input) instead, or modify blendheaders to do that when it's given only 1 input.

### PR DESCRIPTION
**Pull request by jdavies-st**
Resolves #1449

### CODE REVIEW

File: jwst/resample/resample.py [hunk 1]
```diff
@@ -71,6 +71,9 @@ def __init__(self, input_models, output=None, ref_filename=None, **pars):
         self.output_wcs = resample_utils.make_output_wcs(self.input_models)
         log.debug('Output mosaic size: {}'.format(self.output_wcs.data_size))
         self.blank_output = datamodels.DrizProductModel(self.output_wcs.data_size)
+
+        # update meta data and wcs
+        self.blank_output.update(input_models[1])
```
Comment (hbushouse) [inline jwst/resample/resample.py]: Just curious, why index 1 in input_models, rather than 0?

Comment (jdavies-st) [reply jwst/resample/resample.py]: Doh!  *brainfart*  That's a bug.  I should be tagging you to review all my PRs.

Comment (hbushouse) [reply jwst/resample/resample.py]: You actually forgot that python is zero-indexed?! Shame, shame, shame ...

Comment (jdavies-st) [reply jwst/resample/resample.py]: I know.  I should probably stop working right now.

**Pull request merged**